### ISSUE TITLE
(Layout)  - Redmine jsToolbar Images Extension

### ISSUE THREAD

**Issue by denegny** [akabekobeko/redmine-theme-minimalflat2#66]
Use plugin:
https://github.com/tleish/redmine_jstoolbar_ext_images

in plugin:
![screenshot](https://cloud.githubusercontent.com/assets/3710958/13781122/4c245f08-ead3-11e5-85f7-85b4ad516c69.png)

in theme:
![screenshot-area-2016-03-15-171613](https://cloud.githubusercontent.com/assets/3710958/13781134/5812eeba-ead3-11e5-9a8f-a6235c4940ed.png)

**Comment by akabekobeko** [comment]
Memo:

I tried to install the [plugin](https://github.com/tleish/redmine_jstoolbar_ext_images) to the VM on Vagrant.

```
$ cd /var/lib/redmine/plugins
$ git clone https://github.com/tleish/redmine_jstoolbar_ext
$ git clone https://github.com/tleish/redmine_jstoolbar_ext_images
$ cd ..
$ sudo /usr/local/bin/rake redmine:plugins:migrate RAILS_ENV=production
/var/lib/redmine/vendor/bundle/ruby/2.2.0/gems/htmlentities-4.3.1/lib/htmlentities/mappings/expanded.rb:465: warning: duplicated key at line 466 ignored: "inodot"
Migrating redmine_jstoolbar_ext (Redmine jsToolbar Extension)...
Migrating redmine_jstoolbar_ext_images (Redmine jsToolbar Images Extension)...
$ cd tmp
$ sudo touch restart.txt
...Reboot Redmine
```

But plugin does not work with the default theme.
Sub menu does not appear when click on the image button in the issue edit.

**Comment by akabekobeko** [comment]
Memo:

* Become capable of displaying a menu of plugins and add an image to the issue
* Collapses displayed in default theme
* But to fix so as to have the same state as the default theme as minimalflat2

**Comment by akabekobeko** [comment]
@denegny

Thank you for the report, Please check the fixed in v1.1.4.

**Comment by denegny** [comment]
sincere gratitude
